src/ui/app/MfeContainer.js

```diff
@@ -6,23 +6,37 @@ import PropTypes from 'prop-types';
 import { useSelector } from 'react-redux';
 import { getLocale } from 'state/locale/selectors';
 import { getLoggedUserPermissions } from 'state/permissions/selectors';
+import { getDomain } from 'helpers/resourcePath';
+import { getSystemReport } from 'state/system/selectors';
 
 const MfeContainer = ({ id, history }) => {
   const [assetLoading, mfe] = useMfe(id);
   const locale = useSelector(getLocale);
   const permissions = useSelector(getLoggedUserPermissions);
+  const systemReport = useSelector(getSystemReport);
 
   useEffect(() => {
     if (!window.appBuilderRouter) {
       window.appBuilderRouter = history;
     }
-  }, [history]);
+    if (!window.entando || !window.entando.globals) {
```

Comment:
this if condition will prevent the `window.entando` object to get updated(for example, if locale changes), is that what we want?